components/Subscribe/IFTTTItem.js

```diff
@@ -0,0 +1,155 @@
+import { useState, forwardRef, memo } from 'react';
+import { t } from 'ttag';
+import Modal from '@material-ui/core/Modal';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+
+import { makeStyles } from '@material-ui/core/styles';
+import { ButtonIcon } from './FeedDisplayComponents';
+
+import CopyButton from '../CopyButton';
+
+const useStyles = makeStyles(theme => ({
+  paper: {
```

Comment:
If we use [`Dialog`](https://material-ui.com/components/dialogs/) instead of its underlying component [`Modal`](https://material-ui.com/components/modal/), we would not need to define paper style and positioning here. 

Dialogs are not necessary used with `DialogTitle`, `DialogContent` etc, it can be used alone, as shown by `ListPageControls`' `Filters`.